### PR TITLE
System test for nidigital

### LINKED ISSUE #122: Add system tests for NI-Digital session methods
Methods to test include:
get_all_nidigital_instrument_names
set_nidigital_session
get_nidigital_session_for_pattern
get_nidigital_session_for_ppmu
get_all_nidigital_sessions

### PR DESCRIPTION
**Pull request by carmentng**
Resolves #122

### CODE REVIEW

**Review by ni-erikcrank — CHANGES_REQUESTED** [review]

File: systemtests/nidigital_codemodules.py
Comment (smooresni) [reply systemtests/nidigital_codemodules.py:61]: Going to wait until #84 is merged so i can update this PR

File: systemtests/nidigital_codemodules.py
Comment (smooresni) [inline systemtests/nidigital_codemodules.py:83]: @ni-erikcrank can you comment on whether this is the right approach to publishing the number of missing site lists?

**Review by ni-erikcrank — CHANGES_REQUESTED** [review]

File: systemtests/nidigital_codemodules.py
Comment (ni-erikcrank) [inline systemtests/nidigital_codemodules.py:78]: I would omit these lines if possible. They are technically incorrect because they are configuring pins for all sites even though the code should only be affecting the sites in site_list.

**Review by ni-erikcrank — APPROVED** [review]

**Pull request merged**
